spec/models/manageiq/providers/nuage/network_manager/event_catcher/alarm_delete.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Minor, but you might want to move this into a directory named `spec/models/manageiq/providers/nuage/network_manager/event_catcher/data` (we use data as the name in other places).  This way if you add more data files, they are contained.

Reply:
Oh nvm...I thought the specs files and data would be mixed together (which still *might* happen if you create a new thing namespaced under event_catcher), but this *is* the data directory, so this is fine.